### PR TITLE
Update package authors, homepages and descriptions

### PR DESCRIPTION
**Pull request by gavinorland**
Here's a start, at least, on updating these various packages, addressing #54. I've provided basic descriptions of each component - open to change. Also tried to credit authors correctly, but may have this wrong - in particularly may have understated @Loque-'s contributions because I'm not aware of what he did. Open to ideas on this - open to direct commits, too!

### CODE REVIEW

File: components/table-of-contents/package.json [hunk 1]
```diff
@@ -23,5 +22,9 @@
   },
   "main": "lib/index.js",
   "module": "es/index.js",
-  "author": "Steve Sims"
+  "author": "Toby Brancher",
```
Comment (stevesims) [inline components/table-of-contents/package.json]: heh - this one was all me!

File: components/table/package.json [hunk 2]
```diff
@@ -24,9 +24,10 @@
   "main": "lib/index.js",
   "module": "es/index.js",
   "author": "Tarandeep Singh Chauhan",
```
Comment (stevesims) [inline components/table/package.json]: table was originally written by me

File: package.json [hunk 3]
```diff
@@ -146,6 +146,7 @@
     "url": "https://github.com/stevesims/govuk-frederic.git"
   },
   "author": "Steve Sims",
+  "contributors": ["Tarandeep Singh Chauhan", "Gavin Orland", "Toby Brancher"],
```
Comment (stevesims) [inline package.json]: I'd be inclined to have the `author` as `The govuk-frederic authors` and then list us all as contributors

We should also add Alasdair Mcleay, Mark Chambers, Jonathan Sheldrake, and Simon Moore to the contributors list as well

File: components/object-table/package.json [hunk 4]
```diff
@@ -23,10 +23,11 @@
   },
   "main": "lib/index.js",
   "module": "es/index.js",
-  "author": "Gavin Orland",
+  "author": "Toby Brancher",
```
Comment (stevesims) [inline components/object-table/package.json]: this one was me too, with contributions by Toby

File: components/arrow-left/package.json [hunk 5]
```diff
@@ -23,10 +23,11 @@
   },
   "main": "lib/index.js",
   "module": "es/index.js",
-  "author": "Tarandeep Singh Chauhan",
+  "author": "Steve Sims",
```
Comment (stevesims) [inline components/arrow-left/package.json]: I think this one was Mark Chambers

File: components/arrow/package.json [hunk 6]
```diff
@@ -17,10 +17,11 @@
   },
   "main": "lib/index.js",
   "module": "es/index.js",
-  "author": "Tarandeep Singh Chauhan",
+  "author": "Steve Sims",
```
Comment (stevesims) [inline components/arrow/package.json]: not sure who wrote this - probably best to leave it neutral as "The govuk-frederic authors"

File: components/compact-table-accordion-group/package.json [hunk 7]
```diff
@@ -24,10 +24,11 @@
   },
   "main": "lib/index.js",
   "module": "es/index.js",
-  "author": "Tarandeep Singh Chauhan",
+  "author": "Steve Sims",
```
Comment (stevesims) [inline components/compact-table-accordion-group/package.json]: This one was actually Jonathan Sheldrake, based off the `table-accordion-group` which was Alasdair Mcleay

File: components/count/package.json [hunk 8]
```diff
@@ -22,10 +22,11 @@
   },
   "main": "lib/index.js",
   "module": "es/index.js",
-  "author": "Tarandeep Singh Chauhan",
+  "author": "Steve Sims",
```
Comment (stevesims) [inline components/count/package.json]: this is old - has been touched by many - "The govuk-frederic authors" would be more appropriate

File: components/counter-bar/package.json [hunk 9]
```diff
@@ -25,10 +25,10 @@
   },
   "main": "lib/index.js",
   "module": "es/index.js",
-  "author": "Tarandeep Singh Chauhan",
+  "author": "Gavin Orland",
```
Comment (stevesims) [inline components/counter-bar/package.json]: contributions here by Toby and me too

File: components/distraction-free/package.json [hunk 10]
```diff
@@ -24,10 +24,11 @@
   },
   "main": "lib/index.js",
   "module": "es/index.js",
-  "author": "Tarandeep Singh Chauhan",
+  "author": "Steve Sims",
```
Comment (stevesims) [inline components/distraction-free/package.json]: heh - i don't want to claim this ;)

i think Mark Chambers wrote this one

File: components/header-button/package.json [hunk 11]
```diff
@@ -23,10 +23,11 @@
   },
   "main": "lib/index.js",
   "module": "es/index.js",
-  "author": "Tarandeep Singh Chauhan",
+  "author": "Steve Sims",
```
Comment (stevesims) [inline components/header-button/package.json]: i can't remember who exactly wrote this

File: components/keyline/package.json [hunk 12]
```diff
@@ -25,10 +25,11 @@
   },
   "main": "lib/index.js",
   "module": "es/index.js",
-  "author": "Tarandeep Singh Chauhan",
+  "author": "Steve Sims",
```
Comment (stevesims) [inline components/keyline/package.json]: think this one was Simon

File: components/open-button/package.json [hunk 13]
```diff
@@ -22,10 +22,11 @@
   },
   "main": "lib/index.js",
   "module": "es/index.js",
-  "author": "Tarandeep Singh Chauhan",
+  "author": "Steve Sims",
```
Comment (stevesims) [inline components/open-button/package.json]: this one was Ali

File: components/remove-button/package.json [hunk 14]
```diff
@@ -22,10 +22,11 @@
   },
   "main": "lib/index.js",
   "module": "es/index.js",
-  "author": "Tarandeep Singh Chauhan",
+  "author": "Steve Sims",
```
Comment (stevesims) [inline components/remove-button/package.json]: Mark Chambers wrote this

File: components/result-count-title/package.json [hunk 15]
```diff
@@ -23,10 +23,11 @@
   },
   "main": "lib/index.js",
   "module": "es/index.js",
-  "author": "Tarandeep Singh Chauhan",
+  "author": "Steve Sims",
```
Comment (stevesims) [inline components/result-count-title/package.json]: this one might be me - can't remember exactly - plain "The govuk-frederic authors" may be best

File: components/result-count/package.json [hunk 16]
```diff
@@ -23,10 +23,11 @@
   },
   "main": "lib/index.js",
   "module": "es/index.js",
-  "author": "Tarandeep Singh Chauhan",
+  "author": "Steve Sims",
```
Comment (stevesims) [inline components/result-count/package.json]: as with many others, a generic authors may be best here

File: components/spinner/package.json [hunk 17]
```diff
@@ -19,10 +19,11 @@
   },
   "main": "lib/index.js",
   "module": "es/index.js",
-  "author": "Tarandeep Singh Chauhan",
+  "author": "Steve Sims",
```
Comment (stevesims) [inline components/spinner/package.json]: this one should be generic

**Comment by gavinorland** [comment]
Thanks Steve - I'll work my way through these. I put myself down as a contributor quite often btw because I wrote a lot of the tests.

File: package.json (hunk 3 above)
Comment (gavinorland) [reply package.json]: 👍  Have added these in alphabetical order.

**Pull request merged**